calendar-bundle/src/Resources/contao/modules/ModuleEventlist.php

```diff
@@ -314,7 +314,7 @@ protected function compile()
 			$objTemplate->classList = $event['class'] . ' cal_' . $event['parent'];
 			$objTemplate->classUpcoming = $event['class'] . ' cal_' . $event['parent'];
 			$objTemplate->readMore = StringUtil::specialchars(sprintf($GLOBALS['TL_LANG']['MSC']['readMore'], $event['title']));
-			$objTemplate->more = $event['linkText'] ?: $GLOBALS['TL_LANG']['MSC']['more'];
+			$objTemplate->more = $event['linkText'] ?? $GLOBALS['TL_LANG']['MSC']['more'];
```

Comment:
```suggestion
			$objTemplate->more = ($event['linkText'] ?? null) ?: $GLOBALS['TL_LANG']['MSC']['more'];
```
As an empty `linkText` should also fallback to the default text I think?

Reply:
True fixed in 959346256.